client-app/core/api/graphql/cart/mutations/moveFromSavedForLater/index.ts

```diff
@@ -0,0 +1,23 @@
+import { useMutation } from "@vue/apollo-composable";
+import { MoveFromSavedForLaterDocument } from "@/core/api/graphql/types";
+import { globals } from "@/core/globals";
+import type { CartWithListType } from "@/core/api/graphql/types";
+
+export async function moveFromSavedForLater(cartId: string, lineItemIds: string[]) {
+  const { storeId, userId, cultureName, currencyCode } = globals;
+
+  const { mutate: _moveFromSavedForLater } = useMutation(MoveFromSavedForLaterDocument);
+
+  const result = await _moveFromSavedForLater({
+    command: {
+      storeId,
+      userId,
+      cultureName,
+      currencyCode,
+      cartId,
+      lineItemIds,
+    },
+  });
+
+  return result?.data?.moveFromSavedForLater as CartWithListType;
```

Comment:
Try not to cast it to avoid errors. CartWithListType is the full type. In real object from API could be less fields